modules/architecture-machine-roles.adoc

```diff
@@ -26,11 +26,11 @@ Machine sets are groupings of machine resources under the `machine-api` namespac
 [id="defining-masters_{context}"]
 == Cluster masters
 
-In a Kubernetes cluster, the master nodes run services that are required to control the Kubernetes cluster. In {product-title}, the master machines are the control plane. They contain more than just the Kubernetes services for managing the {product-title} cluster. Because all of the machines with the control plane role are master machines, the terms _master_ and _control plane_ are used interchangeably to describe them. Instead of being grouped into a machine set, master machines are defined by a series of standalone machine API resources. Extra controls apply to master machines to prevent you from deleting all master machines and breaking your cluster.
+In a Kubernetes cluster, the control plane nodes (also known as the master nodes) run services that are required to control the Kubernetes cluster. In {product-title}, the control plane machines (also known as the master machines) are the control plane. They contain more than just the Kubernetes services for managing the {product-title} cluster. Because all of the machines with the control plane role are control plane machines (also known as the master machines), the terms _master_ and _control plane_ are used interchangeably to describe them. Instead of being grouped into a machine set, control plane machines (also known as the master machines) are defined by a series of standalone machine API resources. Extra controls apply to control plane machines (also known as the master machines) to prevent you from deleting all control plane machines (also known as the master machines) and breaking your cluster.
```

Comment:
This shouldn't be doing the parentheses so many times. If you need to do it, do it just for the first instance, but not the rest.

Reply:
We should probably have a rule: one parenthesis per paragraph, section, or module. This is definitely repetitive here.